abi-gen/templates/TypeScript/contract.handlebars

```diff
@@ -146,11 +138,7 @@ export class {{contractName}}Contract extends BaseContract {
         {{> typed_params inputs=ctor.inputs}}
     ): Promise<{{contractName}}Contract> {
         assert.isHexString('bytecode', bytecode);
-        assert.doesConformToSchema('txDefaults', txDefaults, schemas.txDataSchema, [
-            schemas.addressSchema,
-            schemas.numberSchema,
-            schemas.jsNumber,
```

Comment:
These seem to be removed entirely, are they automatically resolved now?

Reply:
Yup! They are loaded from @0x/json-schemas when calling `new SchemaValidator()`.